pkg/download/download.go

```diff
@@ -61,6 +61,10 @@ func Download(endpoint string, opts ...Option) (b []byte, err error) {
 		return b, err
 	}
 
+	if u.Scheme == "file" {
```

Comment:
i don't think we need this actually ... if persistence is required, then we will never download the config

Reply:
This is probably safe to have.